ci-operator/config/openshift/kueue-operator/openshift-kueue-operator-release-1.0.yaml

```diff
@@ -178,6 +192,31 @@ tests:
           memory: 500Mi
     test:
     - chain: kueue-operator-test-e2e
+- always_run: false
+  as: test-e2e-disconnected
+  steps:
+    cluster_profile: gcp-observability
+    env:
+      CATSRC_IMG: registry.redhat.io/redhat/redhat-operator-index:v4.19
+      CATSRC_NAME: redhat-operators
+      MIRROR_OPERATORS: "true"
+    test:
+    - chain: kueue-operator-install
+    - as: e2e-kueue
+      cli: latest
+      commands: |
+        # Source proxy configuration if available for disconnected environments
+        if test -f "${SHARED_DIR}/proxy-conf.sh" ; then
+          echo "Setting proxy configuration for e2e tests..."
+          source "${SHARED_DIR}/proxy-conf.sh"
+        fi
+        make test-e2e
+      from: kueue-operator-src
+      resources:
+        requests:
+          cpu: 300m
+          memory: 500Mi
```

Comment:
we could add a post step to collect the must-gather and perhaps remove some of the assets we are collecting for debug in various steps